tools/autograd/derivatives.yaml

```diff
@@ -260,7 +260,7 @@
 
 - name: angle(Tensor self) -> Tensor
   self: angle_backward(grad, self)
-  result: handle_r_to_c(result.scalar_type(), angle_backward(self_t, self_p))
+  result: handle_r_to_c(result.scalar_type(), angle_backward(self_t.conj(), self_p).conj())
```

Comment:
fwiw, the `r_to_c` returns a real tensor, so the final `.conj()` is not needed.

Reply:
ah good catch!